charts/gateway/production-values.yaml

```diff
@@ -1217,7 +1217,7 @@ redis:
     autoGenerated: true
   image:
     registry: docker.io
-    repository: bitnami/redis
+    repository: bitnamilegacy/redis
```

Comment:
i think all places of changing the repo to anything not `bitnami` requires
global:
  security:
    allowInsecureImages: true